qcodes_contrib_drivers/drivers/Attocube/ANC350.py

```diff
@@ -0,0 +1,510 @@
+                           label="Position",
+                           get_cmd=self._get_position,
+                           unit="mm or m°")
+
+        self.add_parameter("frequency",
+                           label="Frequency",
+                           get_cmd=self._get_frequency,
+                           set_cmd=self._set_frequency,
+                           vals=vals.Ints(1, 5000),
+                           unit="Hz")
+
+        self.add_parameter("amplitude",
+                           label="Amplitude",
+                           get_cmd=self._get_amplitude,
+                           set_cmd=self._set_amplitude,
+                           vals=vals.Numbers(0, 70),
+                           unit="V")
+
+        self.add_parameter("status",
+                           label="Status",
+                           get_cmd=self._get_status)
+
+        self.add_parameter("target_position",
+                           label="Target Position",
+                           get_cmd=None,
+                           set_cmd=self._set_target_position,
+                           vals=vals.Numbers(),
+                           unit="mm or m°")
+
+        self.add_parameter("target_range",
+                           label="Target Range",
+                           get_cmd=None,
+                           set_cmd=self._set_target_range,
+                           vals=vals.Numbers(),
+                           unit="mm or m°")
+
+        self.add_parameter("actuator",
+                           label="Actuator",
+                           get_cmd=None,
+                           set_cmd=self._set_actuator,
+                           vals=vals.Numbers(0, 255))
+
+        self.add_parameter("actuator_type",
+                           label="Actuator Type",
+                           get_cmd=self._get_actuator_type)
+
+        self.add_parameter("actuator_name",
+                           label="Actuator Name",
+                           get_cmd=self._get_actuator_name)
+
+        self.add_parameter("capacitance",
+                           label="Capacitance",
+                           get_cmd=self._get_capacitance,
+                           unit="nF")
+
+        if self._parent._version_no >= 4:
+            voltage_get: Optional[Callable] = self._get_voltage
+        else:
+            voltage_get = None
+
+        self.add_parameter("voltage",
+                           label="Voltage",
+                           get_cmd=voltage_get,
+                           set_cmd=self._set_voltage,
+                           vals=vals.Numbers(0, 70),
+                           unit="V")
+
+        self.add_parameter("output",
+                           label="Output",
+                           val_mapping={False: 0,
+                                        True: 1,
+                                        "off": 0,
+                                        "on": 1,
+                                        "auto": 2},
```

Comment:
Could you point out where you found 'auto' that can be mapped to 2?

Reply:
It's described in a comment above: https://github.com/QCoDeS/Qcodes_contrib_drivers/pull/74#discussion_r508315934